metadata-service/configuration/src/main/resources/bootstrap_mcps/data-platforms.yaml

```diff
@@ -757,4 +757,13 @@
     displayName: vertexai
     type: OTHERS
     logoUrl: "/assets/platforms/vertexai.png"
-
+- entityUrn: urn:li:dataPlatform:hex
+  entityType: dataPlatform
+  aspectName: dataPlatformInfo
+  changeType: UPSERT
+  aspect:
+    datasetNameDelimiter: "."
+    name: hex
+    displayName: Hex
+    type: OTHERS
+    logoUrl: "/assets/platforms/hex.png"
```

Comment:
we also need to add this image

Reply:
docs and logo in commit deb4be9d1b2da603cf60b99a254b6e919d0fc42e

Reply:
Not trivial to identify what's expected in the different source doc files: `README.md`, `hex.md`, `hex_pre.md`, ... There should be a template for each of them with the expected content (_eg_ some predefined headers).